bn-models/src/models/trips-api/em-source.ts

```diff
@@ -0,0 +1,9 @@
+import { BaseLookup } from '../_base/base-lookup';
+export const EMSourceTypeName = 'em-source';
+
+export type EMSource = BaseLookup;
```

Comment:
Oh didn't realize we are going to have a lookup doc for this wondering if we still need the enum in catches.ts then? Also just curious was looking at the source types defined in couch already and didn't see NWFSCAudit, are we planning to add that, or we just doing without it?

Reply:
I'm not sure whether the lookups should replace the enum.  Maybe they make sense together.  
There's no NWFSCAudiit lookup as I don't want third parties inadvertently or maliciously creating audits.  My current thinking is we don't expose NWFSCAudit as a valid source option in the tripsApi docs, and don't provide that option in the dropdown on the api portal in boatnet, but the api should accept NWFSCAudit as a valid source (for when NWFSC staff submit an audit).

Is there a way to alias catch-disposition.  Was thinking it's basically the same thing with a different name.  Maybe it makes sense to just create type disposition for EM though.  Thoughts?

Reply:
got it thanks for the auidt explanation. 

there is a disposition enum if that's what you're talking about. Yeah just thinking in general whether to handle these lookups by using enums to docs. Or what general practice we should be following for that. 
https://github.com/nwfsc-fram/boatnet-module/blob/master/bn-models/src/models/_common/enums.ts#L17

Reply:
could make a good group conversation - I don't know what best practice is.